src/PubSub/PubSubClient.php

```diff
@@ -357,6 +357,8 @@ function (array $subscription) {
      * @param Subscription $subscription The subscription to take a snapshot of.
      * @param array $options [optional] Configuration options.
      * @return Snapshot
+     * @experimental
+     * @whitelisted
```

Comment:
Yes, that's the right method. It makes sense that it doesn't break anything, but perhaps it still makes sense to document it as being unavailable, since it can't be used in any way? Similarly for the `Snapshot` class - specific methods are documented as whitelisted, but the whole class is unusable (in the less of being useless, even if it doesn't instantly throw) by someone not on the whitelist.